### DISCUSSION TITLE
SwaggerUI deletion random file in list always removes the last one in the list (e.g. when uploading multiple files to an endpoint)

### DISCUSSION THREAD

**Discussion by nicolastoira** [fastapi/fastapi#9497] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi import FastAPI, UploadFile, File
from typing import List


app = FastAPI()

@app.post("/Upload_files")
async def upload_external_files(files: List[UploadFile] = File(...), ) -> str:
    print("Files uploaded")
```

### Description

* Open the documentation page /docs where SwaggerUI is shown
* The endpoint `/Upload_files` looks like this
![image](https://user-images.githubusercontent.com/48008328/236748222-66bee517-e29e-4ed6-bbcc-f08a7322ccee.png)
* Let's say we would like to upload several files to the endpoint, therefore we list files 1 to 3 as shown. We then realize that the first file we listed (1) is the wrong one. If we click on the deletion sign on the right of the first file, we expect it to be removed from the list. Instead, it is always the last file in the list to be removed and therefore there is no way to remove a specific file in the list:
![image](https://user-images.githubusercontent.com/48008328/236748583-457a36b6-465f-4ce2-a76c-e6550ef9f599.png)

Is this an issue related to FastAPI, or to SwaggerUI? Is there already a fix? 
I updated to the latest FastAPI version, but the issue persists and I'm not sure which version of SwaggerUI is used by FastAPI and if there is already a fix on a latest version of SwaggerUI.

Thanks for the support.

### Operating System

Linux

### FastAPI Version

0.95.1

### Python Version

3.8.10

**Comment by jgould22** [comment, marked as answer]
FastAPI uses the latest version 4 served by a jsdelivr 

https://github.com/tiangolo/fastapi/blob/master/fastapi/openapi/docs.py#LL20C1-L20C1

Likely this is an OpenAPI issue since you can recreate it in the swagger editor fairly easily with something like the below 

```yaml
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                filename:
                  type: array
                  items:
                    type: string
                    format: binary
```

You should probably open an issue over at the swagger ui repo.

**Reply by nicolastoira** [reply]
Thank you @jgould22
